readthedocs/doc_builder/python_environments.py

```diff
@@ -187,7 +187,7 @@ def install_core_requirements(self):
 
         # Use conda for requirements it packages
         requirements = [
-            'sphinx==1.3.1',
+            'sphinx==1.3.5',
```

Comment:
This will break in conda, as it isn't packaging new Sphinx versions. Not sure if we install it with pip, or keep it at 1.3.1